src/main/java/io/confluent/copycat/jdbc/DataConverter.java

```diff
@@ -150,10 +148,10 @@ private static void addFieldSchema(ResultSetMetaData metadata, int col,
       // for single precision
       case Types.FLOAT:
```

Comment:
Good point. This trips me up since i usually think of Float and Double as being different precision, JDBC's Real just makes this all confusing.